pkgs/top-level/all-packages.nix

```
@@ -1451,6 +1451,8 @@ in
 
   clingo = callPackage ../applications/science/logic/potassco/clingo.nix { };
 
+  clingcon = callPackage ../applications/science/logic/potassco/clingcon.nix { };
```

If you'd like to add optional Python and Lua support, it might be a good idea to put 2 flavors of clingcon declared here:

```suggestion
  clingcon = callPackage ../applications/science/logic/potassco/clingcon.nix { };
  
  clingconFull = callPackage ../applications/science/logic/potassco/clingcon.nix {
    enablePython = true;
    enableLua = true;
  };
```